posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -1203,7 +1209,7 @@
       group4_created_at DateTime64,
       person_mode Enum8('full' = 0, 'propertyless' = 1, 'force_upgrade' = 2)
       
-      , _headers Array(Tuple(String, String))
```

Comment:
Oops, looks like this needs to be on migrations for test setup.

(Otherwise all the fixes look good, thank you!)

Reply:
Oops, forgot these were implicit: https://clickhouse.com/docs/engines/table-engines/integrations/kafka#virtual-columns